src/app/individual/individual-detail.component.ts

```diff
@@ -264,10 +263,7 @@ export class IndividualDetailComponent extends BaseIndividualDetailComponent imp
 
             this.observationService
               .upsert(observation, observationId)
-              .pipe(first())
-              .subscribe(_ => {
-                this.updateLastObservation(detail, result.phenophase);
-              });
+              .pipe(first());
```

Comment:
I'm not sure if this works without a subscription. At least the `pipe` `first` doesn't trigger anything. So if it works without the subscription then the `pipe` `first` would also be obsolete.

Reply:
Yes, the upsert works. Removed the pipe as we do not need to subscribe to the returned observable anymore.